### PR TITLE
ENH: attach code snippet number to RUN section

### PR DESCRIPTION
**Pull request by adswa**

### CODE REVIEW

**Comment by adswa** [comment]
ping @mih

File: sphinxcontrib/autorunrecord.py [hunk 1]
```diff
@@ -165,11 +165,13 @@ def get_code(self, encode=False):
...
 
 
-    def write_cast(self, capture_file_cast):
+    def write_cast(self, capture_file_cast, castcount):
         """Write a cast from tagged code examples"""
         import shlex
         notes = self.options.get('notes', None)
         code = self.get_code(encode=False)
+        counter = '### Code snippet {}\n'.format(castcount)
+        code = counter + code
```
Comment (mih) [inline sphinxcontrib/autorunrecord.py]: I would merge the two lines into a single format() call

**Review by mih — APPROVED** [review]

File: sphinxcontrib/autorunrecord.py (hunk 1 above)
Comment (adswa) [reply sphinxcontrib/autorunrecord.py]: oh, yes, that makes sense.

**Comment by adswa** [comment]
Will merge this.

**Pull request merged**